tests/std/tests/P0896R4_views_counted/test.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <span>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+
+template <input_or_output_iterator Iter>
+struct convertible_difference {

```

Comment:
I suggest making this type non-movable or the hard-mode version of non-movable where SMFs `static_assert(always_false<Iter>)`.

Reply:
deleted the move SMFs